package.json

```diff
@@ -11,6 +11,7 @@
         "lint": "eslint .",
         "pr-checks": "pnpm install && pnpm build && pnpm lint && pnpm test && pnpm bench && pnpm build-pages",
         "docgen": "ts-node ./scripts/docgen.ts && pnpm build-pages",
+        "testDocgen": "mocha ./scripts/docgen/test/extractSnippets.test.ts",
```

Comment:
One last thing, let's also update the main `test` command to call this so it runs in CI:

```"test": "pnpm -r test && pnpm test-docgen"```

I suppose we should also update the name of the command to `test-docgen` to match the casing of the other project-level commands (I realize there is probably some inconsistency on this within the repo so that's on me).